src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li,
+a,
```

Comment:
Maybe I wasn't attentive, but why do you reset the styles for these tegs If you don't use them?

Reply:
@official-artem I've just copy my previous code, it was first file and I don't know if i will use them in the code, and I even don't check it again, but good point, thanks!